### PR TITLE
[HOPSWORKS-1871] Add admin DNs to all elasticsearch nodes

### PR DESCRIPTION
**Pull request by maismail**
tests: https://github.com/logicalclocks/hops-testing-ee/pull/112
PR on 1.3: https://github.com/logicalclocks/kagent-chef/pull/160

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -99,4 +99,5 @@
 default["kagent"]["certs"]["host_key"]                  = "#{node["kagent"]["certs_dir"]}/priv.key"
 default["kagent"]["certs"]["elastic_admin_key"]         = "#{node["kagent"]["certs_dir"]}/elastic_admin.key"
 default["kagent"]["certs"]["elastic_admin_certificate"] = "#{node["kagent"]["certs_dir"]}/elastic_admin.pem"
-default["kagent"]["certs"]["elastic_admin_cn"]          = node['install']['localhost'].casecmp?("true") ? "ELkAdmin-localhost" : "ELkAdmin-#{node['fqdn']}"
+default["kagent"]["certs"]["elastic_admin_prefix"]      = "ELkAdmin"
```
Comment (tkakantousis) [inline attributes/default.rb:102]: If we don't want to allow overriding it in the cluster-def, then we can hard-code it in line 103?

**Comment by tkakantousis** [comment]
@maismail open a PR for branch 1.3 as well

File: attributes/default.rb (hunk 1 above)
Comment (maismail) [reply attributes/default.rb:102]: @tkakantousis  I moved into a different attribute to be used in elasticsearch-chef as well 
https://github.com/logicalclocks/elasticsearch-chef/pull/46/files#diff-af441e9eeb4a576380ad3d0def0a4ef0R19

Comment (tkakantousis) [reply attributes/default.rb:102]: Sounds good!

**Comment by maismail** [comment]
@tkakantousis 
https://github.com/logicalclocks/kagent-chef/pull/160

**Comment by maismail** [comment]
With the newly reworked certificates, there is no need for this PR

**Pull request closed without merging**